internal/pkg/template/api/service/error.go

```diff
@@ -0,0 +1,15 @@
+package service
+
+type NotImplementedError struct{}
+
+func (NotImplementedError) ErrorName() string {
+	return "notImplemented"
+}
+
+func (NotImplementedError) Error() string {
+	return "operation not implemented"
+}
+
+func (NotImplementedError) ErrorUserMessage() string {
+	return "Operation not implemented."
+}
```

Comment:
Priklad: tato chyba moze nastat, ale nie je to bezna/validna odpoved, je to interna chyba.
- je mozne definovat co sa ma poslat uzivatelovi, HTTP status tu nie je, tak sa pouzije default `500`.
- a kedze je HTTP code > 499, tak sa to zaloguje ako ERROR do stdout, aj do DD.